src/ui/Avatar/index.tsx

```diff
@@ -12,9 +12,10 @@ const imageRendererClassName = 'sendbird-avatar-img';
 interface DefaultComponentProps {
   width: string | number,
   height: string | number,
+  text?: string,
 }
 
-const DefaultComponent = ({ width, height }: DefaultComponentProps): ReactElement => {
+export const DefaultComponent = ({ width, height, text }: DefaultComponentProps): ReactElement => {
```

Comment:
1. It seems a bit strange that text is being rendered in the avatar component.
2. Using export, the component's name is too generic for practical use.

Reply:
I moved to a new subclass